src/modules/nunjucks/DateFilter.ts

```diff
@@ -0,0 +1,27 @@
+export function dateFilter(value: string ): string {
+    // tslint:disable-next-line: prefer-const
+    let [year, month, day] = value.split('-').map(i => {
+        return parseInt(i, 10);
+    });
+
+    if (!year || !month || !day || month > 11 || month < 0) {
+        throw new Error('Input should be an ISO-formatted date string');
+    }
+
+    const months = [
```

Comment:
This feels unnecessary. Wouldn't it be better to use date formatting library of https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl/DateTimeFormat for example?

Reply:
I attempted something similar to what you're suggesting but it lead to inconsistent output between my local machine and the pipeline. I'll try one of these just in case.

Reply:
Isn't that a matter of locale set on the machine. Maybe pipelines use something different. Have you tried setting locale in constructor .e.g. `new Intl.DateTimeFormat('en-GB')`?

Reply:
I was originally using `Date.toLocaleDateString('en-GB', options)` so was very confused when the pipeline gave a different output. My assumption was that by specifying the locale, the output would be consistent. When I realised this wasn't the case I hard-coded the solution you can see now. I'll try it with the Intl.DateTimeFormat as suggested and see if the output is still inconsistent.

Reply:
Nope, this still isn't working. See screenshot. I have no idea why specifying the locale doesn't lock down the format - it's infuriating.
<img width="660" alt="Screenshot 2020-07-29 at 17 34 43" src="https://user-images.githubusercontent.com/43062389/88827503-f1921f00-d1c1-11ea-9433-9c852e8d563c.png">

Reply:
Alright, due to time constraints I am happy to accept it for now.